Cargo.lock

```diff
@@ -4564,18 +4563,18 @@ checksum = "77af24da69f9d9341038eba93a073b1fdaaa1b788221b00a69bce9e762cb32de"
 
 [[package]]
 name = "openssl-src"
-version = "111.25.0+1.1.1t"
+version = "300.4.2+3.4.1"
```

Comment:
This upgrades OpenSSL to v3.4.1, you need to evaluate its performance and FIPS requirements.

Reply:
Fixed by #18362